roles/common/templates/main.cf.j2

```diff
@@ -681,3 +681,11 @@ readme_directory = /usr/share/doc/postfix-2.10.1/README_FILES
 
 smtp_tls_loglevel = 1
 smtp_tls_security_level = may
+
+{% if handle_pseudo_email %}
+canonical_maps = mysql:/etc/postfix/mysql-pseudo.cf
+remote_header_rewrite_domain = anomail.conext.surfnetlabs.nl 
```

Comment:
This hostname is SURFnet specific. Changing it into a variable makes it usable outside the SURFnet context as well.

Reply:
Uh woops, that's a stupid oversight, as the variable is already defined.  I'll fix this.